storage/backup/backup-cli/src/backup_types/state_snapshot/tests.rs

```diff
@@ -65,4 +65,6 @@ fn end_to_end() {
             .account_state_root_hash,
         state_root_hash,
     );
+
+    drop(rt);
```

Comment:
i see this in a few places. what does it do?

Reply:
it's a mistake, I should do `rt.shutdown_timeout()`. updating.